src/coreclr/jit/target.h

```diff
@@ -333,52 +359,41 @@ static bool operator!=(regMaskTP first, regMaskTP second)
     return (first.getLow() != second.getLow());
 }
 
-static regMaskTP operator~(regMaskTP first)
+#ifdef TARGET_ARM
+static regMaskTP operator-(regMaskTP first, regMaskTP second)
 {
-    regMaskTP result(~first.getLow());
+    regMaskTP result(first.getLow() - second.getLow());
     return result;
 }
 
-#else
-typedef unsigned regMaskTP;
+static bool operator>(regMaskTP first, regMaskTP second)
+{
+    return first.getLow() > second.getLow();
+}
+
+static regMaskTP& operator<<=(regMaskTP& first, const int b)
+{
+    first = first << b;
+    return first;
+}
 #endif
 
+static regMaskTP operator~(regMaskTP first)
+{
+    regMaskTP result(~first.getLow());
+    return result;
+}
```

Comment:
We also ought to do the work to remove/rewrite uses of these if we can (doesn't have to be in this PR)

Reply:
yes, none of these methods are needed, but without it, i will have to make changes in non-lsra code base of doing `.getLow()`, that I wanted to avoid.